### ISSUE TITLE
Tab completion in chat only shows players in the same server, not all proxied players.

### ISSUE THREAD

**Issue by KoenLemmen** [CodeCrafter47/BungeeTabListPlus#408]
I love the fix #384 but tab completion in chat currently only has the players on the server the user is in. It would be nice if all proxied players were in this tab completion list.

**Comment by CodeCrafter47** [comment]
I agree this would be nice. But it's difficult to implement, so I probably won't do it.

**Comment by jewome62** [comment]
Hello,
I'm not a good Java developper.
it's possible to subsribe to this event ? https://github.com/SpigotMC/BungeeCord/blob/master/api/src/main/java/net/md_5/bungee/api/event/TabCompleteEvent.java

And remove BTLP string into suggestion ?

Thanks

**Comment by CodeCrafter47** [comment]
That's not possible. The event is not called for tab completion in chat, as that is handled on the client.
